intranet/apps/auth/decorators.py

```diff
@@ -51,7 +53,10 @@ def inner(*args, **kwargs):
 def reauthentication_required(wrapped):
     def inner(*args, **kwargs):
         request = args[0]  # request is the first argument in a view
-        if request.session.get("reauthenticated", False):
+        if (
+            "reauthenticated_at" in request.session
+            and 0 <= (time.time() - request.session["reauthenticated_at"]) <= settings.REAUTHENTICATION_EXPIRE_TIMEOUT
```

Comment:
There is no validation that `request.session["reauthenticated_at"]` is an integer.

Reply:
Disregard this

Reply:
I already fixed it.